omnigibson/utils/asset_utils.py

```diff
@@ -427,3 +427,5 @@ def encrypt_file(original_filename, encrypted_filename=None, encrypted_file=None
         download_og_dataset()
     elif args.change_data_path:
         change_data_path()
+
+    og.shutdown()
```

Comment:
Why do we need to add this line? A better question to ask is: do we need to add this line to the end of every example?

Reply:
IIRC without this line the process simply hangs or raises an error. Calling shutdown actually terminates the process properly. I should have already added the shutdown call (equivalently, `env.close()`) to the end of all our examples